src/webgl/framebuffer.js

```diff
@@ -187,15 +186,6 @@ export default class Framebuffer extends Resource {
         this._attachTexture({attachment, texture: object, layer: 0, level: 0});
       }
 
-      if (setFilteringParameters && object instanceof Texture2D) {
```

Comment:
Two questions:
- Are there any requirements on filtering parameters in a texture when adding it to a framebuffer? I.e. could attachment fail if the filtering parameters are not correct? My assumption was that filtering has to be set to `GL.NEAREST`...

Note that framebuffer creates default attachments if none are specified. They need the right params.